### DISCUSSION TITLE
Getting Server Error SyntaxError: Unexpected token '.'

### DISCUSSION THREAD

**Discussion by vedpmehta** [heroui-inc/heroui#396] [Help] [answered]
I wanted to add next ui on my next js project.
After following the doc and installation, when i did npm run dev its showing error on browser **Server Error SyntaxError: Unexpected token '.'**

I tried with next UI repo example also, but it is also showing the same error.
Can anyone help me in this.

<img width="982" alt="Screenshot 2022-04-08 at 12 52 27 AM" src="https://user-images.githubusercontent.com/53899376/162280660-aca789bc-5b4b-459a-bf05-124435d6f7b1.png">
<img width="983" alt="Screenshot 2022-04-08 at 12 52 32 AM" src="https://user-images.githubusercontent.com/53899376/162280669-de91932a-872e-4bc0-bd89-0057e6bea638.png">

**Comment by jrgarciadev** [comment, marked as answer]
Hey @vedpmehta could you give more info about your environment?, Node, yarn, npm version etc..

**Reply by vedpmehta** [reply]
HI @jrgarciadev ,
Thanks for responding quickly.
i was using old node and npm version ,
with update of node version it worked, thanks.
